test/typescript/response-generators/client.js

```diff
@@ -174,7 +174,11 @@ async function syncTeam() {
 		text: 'New Event?',
 		user: { id: user1 },
 	});
-	return await client.sync([channel.cid], new Date(Date.now() - 1000 * 60).toISOString());
+	const response = await client.sync([channel.cid], new Date(Date.now() - 1000 * 60).toISOString());
+
+	const serverClient = await utils.disableMultitenancy();
```

Comment:
This won't play well with concurrent runs

Reply:
hmm good point, creating the separate app is the only way then ... for multi-tenancy!!

Reply:
ok just gonna setup different app